trieste/acquisition/rule.py

```diff
@@ -991,7 +991,7 @@ def acquire(
         return thompson_samples
 
 
-class UpdatableTrustRegion(SearchSpace):
+class UpdatableSearchSpace(SearchSpace):
```

Comment:
I moved one method (`get_datasets_filter_mask`) over to `UpdatableTrustRegion` already. The 3 main remaining ones are `initialize`, `update` and `select_in_region` (and some related ones it depends on). So it is just `select_in_region` that we could move, but I thought that it could be more generally useful.

Reply:
Thinking about it some more, I think I will move `select_in_region` to `UpdatableTrustRegion`. Especially since it has "region" in the name.